### ISSUE TITLE
64.2.1: `convert-to-jsdoc-comments` ignores `contexts: ['any']`?

### ISSUE THREAD

**Issue by lkraav** [gajus/eslint-plugin-jsdoc#1747]
Search https://github.com/gajus/eslint-plugin-jsdoc/issues?q=contexts revealed some related discussion but nothing on this specific rule.

I'm trying to avoid hardcoding explicit node type arrays, when I really want literally "everything" to match with `contexts: ['any']`.

Surprisingly, `contexts: ['*']` works to hit everything, but has a side-effect of duplicate hits per site.

EDIT `:not(Program)` selector is somewhat less bad, maybe useful as a data point?

What am I missing?

## Expected behavior
I would expect a regular block comment above `VariableDeclaration` node type to get flagged

## Actual behavior
Block comment above `VariableDeclaration` node type does NOT get flagged

## ESLint Config
```
import jsdoc from 'eslint-plugin-jsdoc';

...

{
    ...
    plugins: {
      jsdoc,
    },
    rules: {
      'jsdoc/convert-to-jsdoc-comments': [
        2,
        {
          contexts: [ 'any' ],
          lineOrBlockStyle: 'block',
        },
      ],
    }
}
```

## ESLint sample

```js
/*
 * Seniority levels that participate in distribution, in display
 * order. `custom` is never a real distribution key. Every test shown in this
 * modal can recruit each of these levels, so all three are always editable.
 */
const SENIORITY_ORDER = ['senior', 'middle', 'specialist'];
```

## Environment

- Node version: 24.19
- ESLint version: 9.39.4
- `eslint-plugin-jsdoc` version: 64.2.1

**Comment by lkraav** [comment]
Added note about `:not(Program)` as a "less blunt than `*`".
